product_pricelist_item_button/models/product_pricelist.py

```diff
@@ -0,0 +1,26 @@
+
+from odoo import fields, models
+
+
+class Pricelist(models.Model):
+    _inherit = 'product.pricelist'
+
+    items_count = fields.Integer(
+        compute='_compute_items_count',
+        string='Pricelist Items')
```

Comment:
Better `Pricelist Items Count`

Reply:
I chose that name because the related field of ids already existed and was called "item_ids", not "pricelist_item_ids"... Then, I wanted to respect the same pattern

Reply:
I'm agree with the field name itself, my comment refers to the label text (the `string` attribute of the field)

Reply:
OK understood. I'm agree. I will change it.